<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewCurve</class>
 <widget class="QDialog" name="NewCurve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add new curve to plot</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLabel" name="label_x">
     <property name="text">
      <string>Select data for X</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_title">
     <property name="text">
      <string>Curve title</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QListWidget" name="listWidget_x"/>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="comboBox_x"/>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="comboBox_y"/>
   </item>
   <item row="4" column="2">
    <widget class="QListWidget" name="listWidget_y"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_y">
     <property name="text">
      <string>Select data for Y</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewCurve</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewCurve</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
